src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryDate.cs

```diff
@@ -324,6 +324,11 @@ public static FormulaValue Date(IRContext irContext, NumberValue[] args)
             var month = (int)args[1].Value;
             var day = (int)args[2].Value;
 
+            if (year <= 1899)
+            {
+                year = 1900 + year;
```

Comment:
I don't think we plan to support this.

Reply:
I'm fine with that, but if we don't then we will need to remove the test that checks this quirk:

>> Date(80, 1, 2)
1/2/1980

Reply:
This test should still work in the TypeScript backend for now. We can just skip that test or provide an alternative answer for just the C# interpreter.

Reply:
I've removed the change and modified the test to have an expected output that matches with the C# implementation.